org.apache.directory.Studio.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.apache.directory.Studio
+
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
```

Comment:
does this really need the GNOME platform?

Reply:
Seems you're right.

Reply:
It actually does need the GNOME Platform for webkit reasons.

```
SWT Webkit: Warning, You are using an old version of webkitgtk. (pre 2.4) BrowserFunction functionality will not be avaliable
SWT WebKit: error initializing DBus server, dBusServer == 0
SWT.CHROMIUM style is not supported in Wayland.
```

With this change the welcome page does not work.